index2.html

```diff
@@ -0,0 +1,30 @@
+    <!DOCTYPE html>
+    <html>
+    <head>
+      <title>Gary's Interest</title>
+    </head>
+    <body>
+    <center>
+      <h1>Gary's Interest</h1>
+    </center>
```

Comment:
@Mvirjilio129 Going forward let try avoid using tags to style you page. Please use css in the future.